dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/FATSuite.java

```diff
@@ -70,5 +70,13 @@ public class FATSuite {
                                     .removeFeatures(new HashSet<>(Arrays.asList(addedFeatures)))
                                     .addFeatures(new HashSet<>(Arrays.asList(ee10AddedFeatures)))
                                     .alwaysAddFeature("servlet-6.0")
+                                    .alwaysAddFeature("jsonb-3.0")
+                                    .conditionalFullFATOnly(FeatureReplacementAction.GREATER_THAN_OR_EQUAL_JAVA_17))
+                    .andWith(FeatureReplacementAction.EE11_FEATURES()
+                                    .removeFeatures(new HashSet<>(Arrays.asList(removedFeatures)))
+                                    .removeFeatures(new HashSet<>(Arrays.asList(addedFeatures)))
+                                    .removeFeatures(new HashSet<>(Arrays.asList(ee10AddedFeatures)))
+                                    .addFeatures(new HashSet<>(Arrays.asList(ee11AddedFeatures)))
+                                    .alwaysAddFeature("servlet-6.1")
                                     .alwaysAddFeature("jsonb-3.0"));
```

Comment:
Do you want a `.conditionalFullFATOnly(FeatureReplacementAction.GREATER_THAN_OR_EQUAL_JAVA_XX))` like you have for the EE9 and 10 repeats?

Reply:
Not in this case. When a build runs the lite mode of a FAT we want it to run with the latest feature set and then run the older features only in FullMode as long as the java version is good. So if we were to run this FAT on Java less than 11 then we would run the EE9 repeat in lite mode since EE10/EE11 require Java 11 / Java17.